### PR TITLE
Cat A Mod 1 RSIS Mappings

### PR DESCRIPTION
**Pull request by Cullima**
# Description and relevant Jira numbers
- Adds data mappings for Cat A Mod 1 test results

## Pull Request checklist

- [ ] [WIP] tag removed from PR title
- [ ] PR has an understandable description

## Git feature branch checklist

- [ ] branch name comply with our branching strategy
- [ ] git branch contains relevant JIRA ticket number
- [ ] branch rebased against the latest develop

## Sign off process checklist

- [ ] Code has been tested manually
- [ ] Tested by QA
- [ ] PO's approval
- [ ] Reviewers selected in Github
- [ ] PR link added to JIRA

### CODE REVIEW

File: src/functions/uploadToRSIS/application/data-mapper/common-mapper.ts [hunk 1]
```diff
@@ -278,3 +278,9 @@ const formatRekeyDateTime = (result: ResultUpload): Date|null => {
   }
   return null;
 };
+
+const formatDrivingSchoolCanidate = (result: ResultUpload): BooleanAsNumber => {
```
Comment (ammarhaiderbjss) [inline src/functions/uploadToRSIS/application/data-mapper/common-mapper.ts]: Typo in `formatDrivingSchoolCanidate` should be `formatDrivingSchoolCandidate`

File: package.json [hunk 2]
```diff
@@ -19,7 +19,7 @@
     "oracledb": "^3.1.2"
   },
   "devDependencies": {
-    "@dvsa/mes-test-schema": "3.19.0",
+    "@dvsa/mes-test-schema": "3.21.1",
```
Comment (szabikr) [inline package.json]: if this goes in the release it shouldn't update to `3.21.*` should stay at `3.20.*`

**Review by rossfauldsbjss — APPROVED** [review]
looks good

File: src/functions/uploadToRSIS/application/data-mapper/__tests__/helpers/cat-a-mod1/inputs/fully-populated-inputs.ts [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+    ...result,
+    testResult: {
+      preTestDeclarations: {
+        insuranceDeclarationAccepted: true,
+        residencyDeclarationAccepted: true,
+        preTestSignature: '**DUMMY**',
+        DL196CBTCertNumber: '123456',
+      },
+      testData: {
+        singleFaultCompetencies: {
```
Comment (szabikr) [inline src/functions/uploadToRSIS/application/data-mapper/__tests__/helpers/cat-a-mod1/inputs/fully-populated-inputs.ts:122]: After the changes that @edwardwilson introduced, the `singleFaultCompetencies` should contain `avoidance` & `emergencyStop` as well as they need to be present in `testData`

**Review by szabikr — APPROVED** [review]
Just a thing that I raised. Not sure it's actually relevant. Otherwise, looks good!

**Pull request merged**